src/events/autoMod.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+
+class CacheManager {
+     constructor(val) {
+          this.cache = String(val).split(/[\n\r]/);
+          this.cache = this.cache.filter((url) => url.length > 0);
+     }
+     hasNot(val) {
+          for (const url of this.cache)
+               if (new RegExp(url).test(val)) return false;
+          return true;
+     }
+}
+
+let cache;
+
+module.exports = {
+     name: "messageCreate",
+     once: false,
+     execute(msg) {
+          if (typeof cache === "undefined")
```

Comment:
I think cache should just be instantiated before the export, and not checked here anymore. seems like unnecessary load to do this check on all messages

Reply:
It's loaded only once, and not at export to prevent overloading at startup